easybuild/easyconfigs/c/COSMO_pompa/patch_COSMO_dom_cray_gpu.patch

```diff
@@ -0,0 +1,83 @@
+diff -Nru cosmo_pompa.orig/cosmo/Options cosmo_pompa/cosmo/Options
+--- cosmo_pompa.orig/cosmo/Options	2018-10-04 10:06:23.000000000 +0200
++++ cosmo_pompa/cosmo/Options	2018-10-04 12:02:30.000000000 +0200
+@@ -1,17 +1,18 @@
+ # compiler
+-F90      = ftn -D__CRAY_FORTRAN__
++F90      = ftn -D__CRAY_FORTRAN__ -h system_alloc 
+ COMPILER = cray
+ 
+ # linker
+ LD       = $(F90)
+ 
+--include $(ROOT)/Options.$(COMPILER).cpu
```

Comment:
Why were they including the CPU options here?

Reply:
This should be what diff finds inside the source-archive.